index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The <code>AudioWorklet</code> object allows importing user-supplied
+          Javascript code that runs on the audio <a>rendering thread</a>. This
+          processing mechanism ensures the synchronous execution of the script
```

Comment:
copy edit for clarity: "...allows developers to supply Javascript to process audio on the rendering thread, supporting custom AudioNodes."